pkg/util/v1alpha2/metricscollector/metricscollector.go

```diff
@@ -0,0 +1,116 @@
+package metricscollector
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
```

Comment:
use Katib-client instead?

Reply:
According to https://stackoverflow.com/questions/53852530/how-to-get-logs-from-kubernetes-using-golang, we cannot get logs without using client-go for now.